client/components/wcc-settings-form/index.js

```diff
@@ -35,7 +35,10 @@ const WCCSettingsForm = ( props ) => {
 		const label = ( currentStepLayout || {} ).action_label || __( 'Next' );
 		return (
 			<FormButtonsBar>
-				<FormButton type="button" onClick={ props.formActions.nextStep }>
+				<FormButton
+					type="button"
+					disabled={ ! currentStepLayout || 0 < props.stepErrors.length || props.form.isSaving }
```

Comment:
Let's make this logic a const like label (I don't like _complicated_ logic in the renders if it can be avoided